required/latex-lab/latex-lab-graphic.dtx

```diff
@@ -137,37 +388,211 @@
-%    \item[\texttt{artifact}] When used the graphic will be tagged as artifact. This doesn't
-%     require a \texttt{BBox} and so works also in some of the not yet supported cases described
-%     above.
-%   \item[\texttt{false}] When used tagging will be stopped completely. It is then the 
+%    \item[\texttt{alt=\meta{text}}] This a second way to tag the graphic as figure with
+%    alternative text. \texttt{alt} without value will tag as figure but not change
+%    the text variable, \texttt{alt=} will empty the text variable (and typically trigger
+%    a warning).
+%    \item[\texttt{actualtext=\meta{text}}] This a second way to tag the graphic as a
+%    symbol with actualtext.  
+%    \item[\texttt{artifact}] This a second way to tag the graphic as artifact. 
+%    \item[\texttt{text}] This switches to text mode.
+%    \item[\texttt{off}] When used tagging will be stopped completely. It is then the 
 %   responsibility of the surrounding code to add appropriate tagging commands.
-%   \item[\meta{name}] Other values will be used as tag names in the structure. If the tag is not
-%   known as a structure tag you will get an warning from tagpdf. The default name is currently
-%   \texttt{Figure}
+%   \item[\texttt{tag=\meta{name}}] This switches to the illustrative mode but uses
+%   \meta{name} as tag name in the structure instead of the default \texttt{Figure}. 
+%   This can for example be used to tag an image of a formula with \texttt{Formula}.
 %   \end{description}   
-%  \item[\texttt{actualtext}] This allows to add an /ActualText to the structure. 
-%  This is useful for small graphics that represent single chars or a short word like a 
-%  logo. If \texttt{actualtext} is used, the graphics is not enclosed in \texttt{Figure}
-%  structure but in a \texttt{Span} structure and no \texttt{/BBox} attribute is added.
-%  This in accordance with (the draft of) PDF/UA-2 but violates perhaps PDF/UA-1.
 %  \item[\texttt{correct-BBox}] If the calculated \texttt{/BBox} values are wrong
-%  they can be correct with this key. It expects four dimensions that are added to the 
+%  they can be corrected with this key. It expects four dimensions that are added to the 
 %  \texttt{/BBox} values. 
-%  \item[\texttt{debug}] The value \texttt{BBox} will show the calculated  \texttt{/BBox} 
-%  as a half transparent red rectangle. 
 %  \end{description}
 %  
-%  The code also redefines the \texttt{alt} key to actually add its values
-%  as an alternative text. If no \texttt{alt} value is given, a warning is issued 
-%  and the file name of the graphic is used.
+%  The code also add to the \texttt{debug} key of \cs{DocumentMetadata} the value \texttt{BBox}.
+%  This adds a half transparent red layer showing the calculated BBox.
+%  The key can only be used after that graphic module has been loaded, which means
+%  one needs a second \cs{DocumentMetadata} call:
+%  \begin{verbatim}
+%   \DocumentMetadata{tagging=on}
+%   \DocumentMetadata{debug=BBox}
```

Comment:
This seems wrong: can't the tag be added earlier even if it is no-op if the graphics code isn't loaded?